src/capi_frontend/capi.cpp

```diff
@@ -660,27 +660,27 @@ OVMS_Status* OVMS_InferenceResponseGetOutput(OVMS_InferenceResponse* res, uint32
     // possibly it is not neccessary to discriminate
     *data = buffer->data();
     *bytesize = buffer->getByteSize();
-    if (spdlog::default_logger_raw()->level() == spdlog::level::trace) {
-        std::stringstream ss;
-        ss << "C-API getting response output of servable: " << response->getServableName()
-           << " version: " << response->getServableVersion()
-           << " output id: " << id
-           << " name: " << *cppName
-           << " datatype: " << toString(ovms::getOVMSDataTypeAsPrecision(*datatype))
-           << " shape: [";
-        size_t i = 0;
-        if (*dimCount > 0) {
-            for (i = 0; i < *dimCount - 1; ++i) {
-                ss << (*shape)[i] << ", ";
-            }
-            ss << (*shape)[i];
-        }
-
-        ss << "]"
-           << " bufferType: " << *bufferType
-           << " deviceId: " << *deviceId;
-        SPDLOG_TRACE(ss.str());
-    }
+    // if (spdlog::default_logger_raw()->level() == spdlog::level::trace) {
+    //     std::stringstream ss;
+    //     ss << "C-API getting response output of servable: " << response->getServableName()
+    //        << " version: " << response->getServableVersion()
```

Comment:
Make Inference response hold copy of name instead of reference to name from request.